lib/src/main/java/com/basistheory/elements/service/ApiClientProvider.kt

```diff
@@ -10,7 +11,8 @@ import kotlinx.coroutines.Dispatchers
 
 internal class ApiClientProvider(
     private val apiUrl: String = "https://api.basistheory.com",
-    private val defaultApiKey: String? = null
+    private val defaultApiKey: String? = null,
+    private val environment: String? = "production"
```

Comment:
We could make an ApiClientProviderV2 or something and only allow people to pass environment and api key, to remove api url if we wanted, though this is consistent with how it is on web elements.